wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -8,46 +8,62 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
-
++ **Do not qualify beatmapsets whose song already has a Qualified beatmapset, unless no game modes of both sets overlap - this includes remixed versions**. For example an osu!taiko and osu! beatmapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
++ **Make sure no mod posts in the beatmap thread have been ignored before nominating a beatmap.** Posts without a sufficient reply may count as ignored. When in doubt, double check if the Code of Conduct for Mapping and Modding has been violated on the beatmap thread and act accordingly.
 
-### Conduct
+## Conduct
 
 + **Do not follow any unfinalized rules or amendments.**
++ **Abide by the [osu! Community Rules](/wiki/Rules) and the [Code of Conduct for Modding and Mapping](/wiki/Ranking_Criteria/Code_of_Conduct) at all times.** As a Beatmap Nominator, your behaviour should serve as a positive example to other members of the community. Infringing upon the Code of Conduct may result in you being placed on a temporary probation, or in extreme cases, removal from the Beatmap Nominators.
 + **Do not nominate a beatmap if you are unsure of something.** In such cases, ask fellow nominators for assistance before making any rushed decisions. This includes unfamiliar game modes, suspiciously gray area/unrankable issues, quality issues or complex timing.
-+ **Do not nominate a beatmap if you cannot reasonably judge it.** Being able to reasonably evaluate a beatmap, either through playtesting or modding experience, is a core part of being able to judge a beatmap’s quality. If you are deemed not to have sufficient playing skill or modding experience, we may call your ability to judge the map into question.
-+ **Do not qualify a beatmap if any significant changes were made since it was bubbled.** These may include fixing unrankable issues, changes in object snapping, major changes in flow or object placement, alterations to a song’s timing, etc. When in doubt, such beatmaps must be rebubbled first.
++ **Do not nominate a beatmap if you cannot reasonably judge it.** Being able to reasonably evaluate a beatmap, either through playtesting or modding experience, is a core part of being able to judge a beatmap’s quality. If you are deemed not to have sufficient playing skill or modding experience, we may call your ability to judge the beatmap into question.
 + **Ensure that kudosu is awarded to all posts that deserve it.** Additionally consult the mapper before giving out kudosu to make sure there are not any reasons why the post does not need kudosu (e.g. the modder has received one already for a different post which you overlooked). If the modpost is too old to receive kudosu, consult the modder to make an additional post linking the original he receives kudosu for.
```

Comment:
this rule kinda makes no sense in mv2, so would remove it anyway. it's written for forum scenarios